src/components/errorScreen/errorScreen.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Grid } from 'patternfly-react';
+
+class ErrorScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { show: true };
+  }
+
+  onTryAgain = () => {
+    window.location.href = '/';
+  };
+
+  render() {
+    if (!this.state.show) {
+      return '';
+    }
+
+    return (
+      <React.Fragment>
+        <Grid.Row className="integr8ly-errorscreen-backdrop">
```

Comment:
This should match the template from `loadingScreen.js`, where the Bootstrap grid is replaced with `display: flex`. This allows for a natural layout with minimal forced positioning.